aim/cli/up/utils.py

```diff
@@ -27,3 +32,119 @@ def build_uvicorn_command(host, port, num_workers, ssl_keyfile, ssl_certfile):
         cmd += ['--ssl-certfile', ssl_certfile]
     cmd += ['aim.web.run:app']
     return cmd
+
+
+def parse_tf_events(tf_logs, repo_inst):
+    """
+    This function scans and collects records from TF event files.
+
+    Creates and uses cache file "tf_logs_cache" in the repo dir
+    to track previously processed files and values
+
+    Currently, supports the following TF plugins
+        - scalar
+        - image
+    """
+
+    # This import statement takes long to complete
+    if not os.path.isdir(tf_logs):
+        logger.error('Value of "tf_logs" should be a valid path to the root of TF logdir.')
+        return
+
+    try:
+        import tensorflow as tf
+        from tensorflow.python.summary.summary_iterator import summary_iterator
+    except ModuleNotFoundError:
+        logger.error('Cannot process TF events - could not import required modules.')
+        return
+
+    tf_logs_cache_path = f'{repo_inst.path}/tf_logs_cache'
```

Comment:
(Very minor thing here) Please `path.join()` instead.

Reply:
how i missed that 🤦‍♂️